peerprep/frontend/src/views/CollabServiceViews/CollabServiceIntegratedView.tsx

```diff
@@ -4,18 +4,19 @@ import { useParams, useNavigate } from 'react-router-dom';
 import 'codemirror/lib/codemirror.css';
 import 'codemirror/theme/material.css';
 import 'codemirror/mode/javascript/javascript';
+import axios from 'axios';
 
 import * as Y from 'yjs';
 import { WebsocketProvider } from 'y-websocket';
-
-// @ts-check
```

Comment:
You can add this line back, it's not a comment but actually a macro (?) to force type-checking. This is because CodemirrorBinding is typeless in javascript, and we want to use it in a typed context like TypeScript